code/modules/mob/living/brain/posibrain.dm

```diff
@@ -181,6 +219,14 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	brainmob.real_name = brainmob.name
 	brainmob.forceMove(src)
 	brainmob.container = src
+
+	//If we are on the station level, add it to the list of available posibrains.
+	var/turf/currentturf = get_turf(src)
+	if( is_station_level(currentturf.z))
+		GLOB.on_station_posis += src
+		var/datum/job/job = SSjob.GetJob(JOB_NAME_POSIBRAIN)
+		job.total_positions = length(GLOB.on_station_posis)
```

Comment:
the codes in `onTransitZ()` checking positron z and manipulating glob list can be merged in a proc.